NEWS.md

```diff
@@ -0,0 +1,53 @@
+# Changes by Version
+
+## [Unreleased]
+
+### Added
+
+* Add `tmpdir` parameter to specify a custom temp directory (#230)
```

Comment:
I propose we do both. This way the person tagging the release is not on the hook to gather the release notes, it should be gathered as changes are added.